doc/whats_new/v0.20.rst

```diff
@@ -46,6 +46,16 @@ Changelog
   data, which raised an error in 0.20.0, now returns the expected output again.
   :issue:`12699` by `Joris Van den Bossche`_.
 
+:mod:`sklearn.metrics`
+......................
+
+- |Fix| Fixed a bug in :func:`metrics.pairwise_distances` and
+  :func:`metrics.pairwise_distances_chunked` where parameters of
+  ``"seuclidean"`` and ``"mahalanobis"`` metrics need to be pre-computed
+  before the data is split into chunks. :issue:`12701` by
```

Comment:
maybe also mention the current behavior, i.e., compute these parameters after the data is splitted.